### PR TITLE
New Workflow: IVM Top Remediations to Jira

### PR DESCRIPTION
**Pull request by sengleson-r7**
New ICON workflow: fetch top remediations from VM, create/update Jira tickets

## Requirements

Developers, verify you have completed the following items by checking them off:

### Checklist
- [x] Workflow/Snippet contains title and detailed description
- [x] Workflow/Snippet title should follow general [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [x] User friendly step names in [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [x] User friendly step descriptions where needed
- [x] Workflow contains at least 1 artifact with well formatted information
- [x] Directory created and named after the title the snippet or workflow e.g. `workflows/Basic_Domain_Enrichment_Report/`, `snippets/Domain_Enrichment`
- [x] Workflow `.icon` file is in the workflow's directory e.g. `workflows/Basic_Domain_Enrichment_Report/Basic_Domain_Enrichment_Report.icon`
- [x] Take PNG screenshots of the workflow or snippet e.g.  https://market.komand.com/workflows/ross/chatops-github-commands/1.0.0, https://market.komand.com/snippets/jschipp/basic-domain-enrichment-report/0.1.0
- [x] Create a directory called `screenshots` in your workflow's/snippet's directory
- [x] Includes PNG screenshots of the workflow in canvas, job, artifact in the `screenshots` directory e.g.  `workflows/Basic_Domain_Enrichment_Report/screenshots/`
- [x] PNG screenshot file names should conform to style e.g. `workflow1.png`, `artifact1.png`, `job.png`
- [x] Workflow verified as working through WUI
- [x] Credentials are not exported and stored in git
- [x] `workflow.spec.yaml` file with metadata accompanies workflow
- [x] Workflow documentation is provided in `help.md`
- [x] Directory structure is followed, see https://github.com/rapid7/insightconnect-workflows/blob/master/README.md

### CODE REVIEW

File: workflows/IVM_Top_Remediations_to_JIRA/help.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+# Key Features
+
+* Creates issues in JIRA for the top 25 solutions from InsightVM
+* Updates issue for a solution with current impacted assets if issue already exists
+
+# Requirements
+
+API and account credentials for
+* InsightVM
```
Comment (zyoutz-r7) [inline workflows/IVM_Top_Remediations_to_JIRA/help.md:14]: Do we want to call out `InsightVM Console`?

File: workflows/IVM_Top_Remediations_to_JIRA/workflow.spec.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+products: ["insightconnect"]
+name: IVM_Top_Remediations_to_JIRA 
+title: "IVM Top Remediations to JIRA"
+description: This workflow identifies the current top remediations from InsightVM, determines if issues have been created in JIRA for the identified remediations, and creates or updates issues for the identified remediations as appropriate.
+version: 1.0.0
+vendor: rapid7
+support: rapid7
+status: []
+hub_tags:
+  use_cases: [vulnerability_management, insightvm, jira]
```
Comment (zyoutz-r7) [inline workflows/IVM_Top_Remediations_to_JIRA/workflow.spec.yaml:11]:
```suggestion
  use_cases: [vulnerability_management]
```

File: workflows/IVM_Top_Remediations_to_JIRA/workflow.spec.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
+extension: workflow
+products: ["insightconnect"]
+name: IVM_Top_Remediations_to_JIRA 
+title: "IVM Top Remediations to JIRA"
```
Comment (zyoutz-r7) [inline workflows/IVM_Top_Remediations_to_JIRA/workflow.spec.yaml:4]:
```suggestion
title: "InsightVM Top Remediations to JIRA"
```

File: workflows/IVM_Top_Remediations_to_JIRA/workflow.spec.yaml [hunk 4]
```diff
@@ -0,0 +1,37 @@
+extension: workflow
+products: ["insightconnect"]
+name: IVM_Top_Remediations_to_JIRA
```
Comment (zyoutz-r7) [inline workflows/IVM_Top_Remediations_to_JIRA/workflow.spec.yaml:3]:
```suggestion
name: InsightVM_Top_Remediations_to_JIRA 
```

And move to updated directory to reflect too.

File: workflows/IVM_Top_Remediations_to_JIRA/workflow.spec.yaml [hunk 5]
```diff
@@ -0,0 +1,37 @@
+extension: workflow
+products: ["insightconnect"]
+name: IVM_Top_Remediations_to_JIRA 
+title: "IVM Top Remediations to JIRA"
+description: This workflow identifies the current top remediations from InsightVM, determines if issues have been created in JIRA for the identified remediations, and creates or updates issues for the identified remediations as appropriate.
```
Comment (zyoutz-r7) [inline workflows/IVM_Top_Remediations_to_JIRA/workflow.spec.yaml:5]: I think we might want to shorten this a bit and let the README go into details.  Perhaps `Creation of Jira issues based on InsightVM top remediations` or something like that?

File: workflows/IVM_Top_Remediations_to_JIRA/help.md (hunk 1 above)
Comment (sengleson-r7) [reply workflows/IVM_Top_Remediations_to_JIRA/help.md]: We've phrased the VM requirements as "* InsightVM Console URL and account credentials" in other workflows

File: workflows/IVM_Top_Remediations_to_JIRA/workflow.spec.yaml (hunk 5 above)
Comment (sengleson-r7) [reply workflows/IVM_Top_Remediations_to_JIRA/workflow.spec.yaml]: Good call -- changed to "Create and update Jira issues based on top risk-reducing remediation steps from InsightVM."

**Pull request merged**